### ISSUE TITLE
[Window Terimal Preview - Tearout tab] : User is not able to perform the 'Tearout', 'Transfer' tab functionality using keyboard.

### ISSUE THREAD

**Issue by Saiteja341** [microsoft/terminal#15156]
### Windows Terminal version

1.18.902.0

### Windows build number

25324.1011

### Other Software

**Test Environment:** 
OS: Windows 11 Version Dev (OS Build 25324.1011)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

Repro Steps:

1.Open Windows Terminal.
2.Open multiple 'Command prompt' using 'Ctrl + Shift + 1' keys.
3.Now try to perform 'Tearout' and 'Transfer' tab functionality using keyboard.
4.Observe the issue.

Users Experience:
Users who rely on keyboard will get impacted here & fail to use the functionality effectively.

Guideline link:
https://www.w3.org/TR/WCAG21/#keyboard

Attachments:
[User is not able to perform the 'Tearout', 'Transfer' tab functionality using keyboard..zip](https://github.com/microsoft/terminal/files/11197147/User.is.not.able.to.perform.the.Tearout.Transfer.tab.functionality.using.keyboard.zip)

### Expected Behavior

User should be able to perform the 'Tearout', 'Transfer' tab functionality using keyboard.
> Even shortcut should be defined to perform 'Tearout', 'Transfer' tab- functionality.

### Actual Behavior

User is not able to perform the 'Tearout', 'Transfer' tab functionality using keyboard.
> Even no shortcut is defined for 'Tearout', 'Transfer' tab- functionality

**Comment by zadjii-msft** [comment]
@Saiteja341 The user is free to define actions like:

```json

        { "keys": "f1", "command": { "action": "moveTab", "window": "1" }, },
        { "keys": "f2", "command": { "action": "moveTab", "window": "2" }, },
        { "keys": "f3", "command": { "action": "movePane", "window": "1" }, },
        { "keys": "f4", "command": { "action": "movePane", "window": "2" }, },
```

in their settings, which would allow them to move tabs / panes to other windows using just the keyboard. Is that sufficient?

**Comment by Saiteja341** [comment]
Hi @zadjii-msft ,

Thanks for providing these command but what if a user doesn't know about these commands, they will not able to perform this functionality.

It should be either pre-define in the terminal's action, So that user might we able to use this functionality with keyboard shortcuts.

**Comment by zadjii-msft** [comment]
@Saiteja341 Would a default action for "move this tab/pane to a new window" be sufficient here? It should be fairly trivial for us to add a pair of 

```
        { "command": { "action": "moveTab", "window": "new" }, },
        { "command": { "action": "movePane", "window": "new" }, },
```

default actions, and have them just work out of the box.

**Comment by zadjii-msft** [comment]
@Saiteja341 gentle ping ☺️ Would a default action for "move this tab/pane to a new window" be sufficient here?

**Comment by zadjii-msft** [comment]
We added the above actions in #15258. If those are insufficient, OP can feel free to reopen.
